### PR TITLE
[OSCI] Add willie-the-lord as new author 

### PR DESCRIPTION
**Pull request by Willie-The-Lord**
### Description

Add willie-the-lord as new author 
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
 
[X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/wiilie_the_lord.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Sung-Jie Hung
+short_name: Willie
+twitter:
```
Comment (ashwin-pc) [inline _authors/osci_2023/wiilie_the_lord.markdown:4]: Can you remove this line?

**Review by ashwin-pc — CHANGES_REQUESTED** [review]

File: _authors/osci_2023/wiilie_the_lord.markdown (hunk 1 above)
Comment (Willie-The-Lord) [reply _authors/osci_2023/wiilie_the_lord.markdown:4]: Ok no prob.

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**